packages/repository/test/acceptance/has-many-without-di.relation.acceptance.ts

```diff
@@ -132,7 +132,10 @@ describe('HasMany relation', () => {
     Customer,
     typeof Customer.prototype.id
   > {
-    public orders: HasManyRepositoryFactory<Order, typeof Order.prototype.id>;
+    public orders: HasManyRepositoryFactory<
+      Order,
+      typeof Customer.prototype.id
```

Comment:
👍 

Here you are using `HasManyRepositoryFactory` with correct types (source model, target id). Let's fix the documentation too. (See one of my earlier comments above.)